### DISCUSSION TITLE
Does this tool support Redhat?

### DISCUSSION THREAD

**Discussion by zhenningx** [CiscoDevNet/iPSK-Manager#53] [General]
I installed iPSK-Manager on RHEL 9. After I login admin portal and click Portals > Add Portals. Nothing happens. No errors in System Logging. Any ideas?

**Comment by ciesinsn** [comment]
Yes it should.  Make sure to enter the other information in the system before trying to create a portal.  Work backwards on the menu.

**Comment by ciesinsn** [comment]
One thing to note for others is I was informed that RHEL does not include libsodium and iPSK Manager uses mod-sodium in PHP which requires the library.  This issue would come up during installation though as that is when libsodium is first used within the code.   So when using RHEL you need to get libsodium from someplace other then RedHat.
